### DISCUSSION TITLE
0022 XLS-22d: rippled API Versioning

### DISCUSSION THREAD

**Discussion by intelliot** [XRPLF/XRPL-Standards#54] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by WietseWind** [comment]
Yes! Great idea. Well thought through, backwards compatible: I love it!

**Comment by mvadari** [comment]
@intelliot should this spec be moved into PR and kept in the repo or should it be closed? Note that this does not mean that the spec has been accepted/implemented/finalized.

I will move it to PR by copy-pasting the existing draft if there is no response in the next week.
